ui/scss/_gui.scss

```diff
@@ -61,6 +61,10 @@ main.main--refreshing {
   }
 }
 
+.icon_set {
```

Comment:
This is a property of a card, or otherwise something like `icon_set--right`.

Icon set CSS can go in `_icons.scss`.

Reply:
I was thinking the right justification would just be the default style and we can override it if we ever use it somewhere other than a card. But you're right, it should be either a modifier or part of `IconSet`.